devicetypes/smartthings/zwave-smoke-alarm.src/zwave-smoke-alarm.groovy

```diff
@@ -21,7 +21,7 @@ metadata {
 
 		attribute "alarmState", "string"
 
-		fingerprint mfr:"0138", prod:"0001", model:"0002", deviceJoinName: "First Alert Smoke Detector and Carbon Monoxide Alarm (ZCOMBO)"
+		fingerprint mfr:"0138", prod:"0001", model:"0003", deviceJoinName: "First Alert Smoke Detector and Carbon Monoxide Alarm (ZCOMBO)"
```

Comment:
Thanks for the PR. Can you leave the existing fingerprint and just add the new one?